code/pizzabot.js

```diff
@@ -1,7 +1,93 @@
+// Variables
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
 
+// Validation
+const pizzaList = [
+  "Vegetarian Pizza",
+  "Hawaiian Pizza",
+  "Pepperoni Pizza",
+  "Vegetarian",
+  "Hawaiian",
+  "Pepperoni",
+  "vegetarian pizza",
+  "hawaiian pizza",
+  "pepperoni pizza",
+  "vegetarian",
+  "hawaiian",
+  "pepperoni", ]
+
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+const validateOrderName = (orderName) => {
+  if (pizzaList.includes(orderName)) {
+    return true;  
+  } else {
+    return false; 
+  }
+};
+
+
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+  const totalCost = (orderQuantity * pizzaPrice);
+  return totalCost;
+};
+
+const calculateCookingTime = (orderQuantity) => {
+
+  if (orderQuantity >= 1 && orderQuantity <= 2 ) {
+    return "10 min";
+
+  } else if (orderQuantity >= 3 && orderQuantity <= 5) {
+    return "15 min";
+
+  } else { 
+    return "20 min";
+  }
+
+};
```

Comment:
I like the thought with returning "xx min".  Good attention to detail.  However, it's always good to set some boundaries for your functions.  A real world example might be:
```javascript
// Returns cooking time based on orderQuantity, in minutes
const calculateCookingTime = (orderQuantity) => {
     // ...
     return 15;
     // ...
}

const cookTime = calculateCookingTime(quantity);
const formattedCookingTime = formatTime(cookTime);
alert(formattedCookingTime); // output
```


By maintaining a number result from `calculateCookingTime` we reserve the possibility to format the time any way we see fit afterwards -- think different languages, seconds, etc